src/frequenz/sdk/actor/_actor.py

```diff
@@ -44,13 +44,33 @@ class Actor(BackgroundService, abc.ABC):
         This is mostly used for testing purposes and shouldn't be set in production.
     """
 
+    def __init__(self, *, name: str | None = None) -> None:
+        """Create actor instance.
+
+        Args:
+            name: The name of this background service.
+        """
+        super().__init__(name=name)
+        self._is_cancelled = False
```

Comment:
Can't we save the main actor task instead and use `self._main_task.cancelled()` as a single source of truth?